### ISSUE TITLE
PyInstaller issues with pycryptodome 3.8.1: Cannot load native module 'Cryptodome.PublicKey._ec_ws'

### ISSUE THREAD

**Issue by SomberNight** [Legrandin/pycryptodome#286]
Windows binary built with pyinstaller 3.4 raises when packaging pycryptodomex 3.8.1.
Issue not present with pycryptodomex 3.7.3.

```
> .\electrum-3.3.4-189-gb11cb11d-dirty.exe
Traceback (most recent call last):
  File "site-packages\dns\dnssec.py", line 482, in <module>
ModuleNotFoundError: No module named 'Crypto'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "run_electrum", line 79, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "c:\python3\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "electrum\__init__.py", line 3, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "c:\python3\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "electrum\wallet.py", line 63, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "c:\python3\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "electrum\contacts.py", line 29, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "c:\python3\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "electrum\dnssec.py", line 42, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "c:\python3\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\dns\dnssec.py", line 489, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "c:\python3\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\Cryptodome\Signature\DSS.py", line 42, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "c:\python3\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\Cryptodome\PublicKey\ECC.py", line 90, in <module>
  File "site-packages\Cryptodome\Util\_raw_api.py", line 300, in load_pycryptodome_raw_lib
OSError: Cannot load native module 'Cryptodome.PublicKey._ec_ws': Trying '_ec_ws.cp36-win32.pyd': Failed to load dynlib/dll 'C:\\Users\\User\\AppData\\Local\\Temp\\_MEI234602\\Cryptodome\\Util\\..\\PublicKey\\_ec_ws.cp36-win32.pyd'. Most probably this dynlib/dll was not found when the application was frozen., Trying '_ec_ws.pyd': Failed to load dynlib/dll 'C:\\Users\\User\\AppData\\Local\\Temp\\_MEI234602\\Cryptodome\\Util\\..\\PublicKey\\_ec_ws.pyd'. Most probably this dynlib/dll was not found when the application was frozen.
[15332] Failed to execute script run_electrum
```

Maybe hook needs updating?
https://github.com/pyinstaller/pyinstaller/blob/06f7da789f41a21dd697478ca317c858bc6be9e4/PyInstaller/hooks/hook-Cryptodome.py#L28-L34

**Comment by Legrandin** [comment]
You are correct. The hook in PyInstaller needs the update describer in the PR https://github.com/pyinstaller/pyinstaller/pull/4229.

**Comment by Legrandin** [comment]
This is now fixed with PyInstaller v3.5 (https://pyinstaller.readthedocs.io/en/v3.5/CHANGES.html).
